content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/windows-endpoint-slice-proxying.md

```diff
@@ -0,0 +1,19 @@
+---
+# Removed from Kubernetes
+title: WindowsEndpointSliceProxying
+content_type: feature_gate
+
+_build:
+  list: never
+  render: false
+---
+
+<!--
+When enabled, kube-proxy running on Windows will use
+EndpointSlices as the primary data source instead of Endpoints, enabling scalability and
+performance improvements. See
+[Enabling Endpoint Slices](/docs/concepts/services-networking/endpoint-slices/).
+-->
+当启用时，运行在 Windows 上的 kube-proxy
+将使用 EndpointSlices 而不是 Endpoints 作为主要数据源，从而实现可伸缩性并改进性能。
```

Comment:
(perhaps for a separate PR?)
```suggestion
将使用 EndpointSlice 而不是 Endpoints 作为主要数据源，从而实现可伸缩性并改进性能。
```

There are two relevant APIs here:
- Endpoints (sic)
- EndpointSlice